diff -rupN jack_capture-20230104.old/Makefile jack_capture-20230104/Makefile
--- jack_capture-20230104.old/Makefile	2023-01-04 06:53:14.000000000 +0100
+++ jack_capture-20230104/Makefile	2025-08-14 12:10:23.256627742 +0200
@@ -1,4 +1,4 @@
-PREFIX ?= /usr/local
+PREFIX ?= /usr
 bindir = $(PREFIX)/bin
 
 VERSION=0.9.73
@@ -12,8 +12,8 @@ CFLAGS += -I/opt/local/include
 OPTIMIZE=-O3
 #OPTIMIZE=-O0 -g
 
-COMPILEFLAGS=$(CFLAGS) $(OPTIMIZE) -DVERSION=\"$(VERSION)\" -Wall -Wextra -Wno-unused
-LINKFLAGS=$(LDFLAGS) -ljack -lsndfile -lm -lpthread -latomic
+COMPILEFLAGS=$(CFLAGS) $(OPTIMIZE) -DVERSION=\"$(VERSION)\" -Wall -Wextra -Wno-unused -fPIE
+LINKFLAGS=$(LDFLAGS) -ljack -lsndfile -lm -lpthread -latomic -pie
 
 OS := $(shell uname)
 ifneq ($(OS),Darwin)
